CCVA_Report.Rmd

```diff
@@ -811,8 +884,13 @@ data. It represents the median change between pairs of data points over time. A
 positive Sen's slope indicates an upward trend, and a negative Sen's slope
 indicates a downward trend. The value of the slope signifies the magnitude of
 the change. The following table of Sen's slopes provides an indication of
+<<<<<<< HEAD
```

Comment:
I see a lot of this in the report edits, is this because there were merge conflicts at some point? Could you remove them before merging? (not an immediate need though).